cost-analyzer/templates/kubecost-cluster-controller-template.yaml

```diff
@@ -256,7 +256,6 @@ spec:
         ports:
         - name: http-server
           containerPort: 9731
-          hostPort: 9731
```

Comment:
@alexkubecost I can dig here, but do you know off hand why we would need hostPort?

I don't understand why anything would be connecting to the clusterController that doesn't know the service name.

Reply:
I can't currently think of any reason why hostPort would be needed. @ameijer ?

Reply:
I'll bet you that this yaml was written from an example where they exposed an http port on a node and the ingress controller routed to that. I agree there is rarely if ever a use case for a host port on non daemon set controllers. I think getting rid of it is a great idea